Makefile

```
@@ -207,10 +207,13 @@ CGO_ENABLED := 0
 endif
 export CGO_ENABLED
 
+export GIT_REPO := github.com/operator-framework/operator-controller
```

Oh one more think I noticed: This technically needs to be to go module name in order to be used within the `$(VERSION_PATH)` variable. So in that case, can we be explicit (and more copy/paste friendly) with this:
```suggestion
export GIT_REPO := $(shell go list -m)
```